lib/manageiq/providers/vmware/engine.rb

```diff
@@ -3,6 +3,10 @@ module Providers
     module Vmware
       class Engine < ::Rails::Engine
         isolate_namespace ManageIQ::Providers::Vmware
+
+        def self.plugin_name
+          _('VMware Provider')
```

Comment:
Hm there are technically two providers in this plugin

Reply:
This string is mainly to be presented in the about dialog in classic UI, so I guess we don't have to be super-precise about it. Though if there's a better way to name it, I'm open to suggestions.